tad/services/storage.py

```diff
@@ -0,0 +1,32 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+
+from yaml import dump
+
+
+class WriteService(ABC):
+    def __init__(self, location: str, filename: str) -> None:
```

Comment:
I would move this init to the fileSystemWriter since it is specific to the writer class

Reply:
agree and it done